### PR TITLE
Add typings for window.ResizeObserver (resize-observer-browser)

### PR DESCRIPTION
**Pull request by chivesrs**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: N/A, `ResizeObserver` is a global property so accessible on `window`
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/resize-observer-browser/index.d.ts [hunk 1]
```diff
@@ -4,14 +4,20 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.7
 
+declare global {
+    interface Window {
```
Comment (sheetalkamat) [inline types/resize-observer-browser/index.d.ts]: You want to add `YourLibraryName-global.test.ts` in the tests as well to verify the globals are working as intended
Refer https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/resize-observer-browser/index.d.ts (hunk 1 above)
Comment (chivesrs) [reply types/resize-observer-browser/index.d.ts]: Done, added into a single commit.

**Comment by chivesrs** [comment]
How do I mark it as no longer author attention needed? Seems like a weird race condition on the typescript-bot.

File: types/resize-observer-browser/test/resize-observer-global.test.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function resizeObserverCreatesViaWindow(): void {
+    const resizeObserver = new window.ResizeObserver((entries) => {
+        const div = document.getElementById('display-div')!;
```
Comment (sheetalkamat) [inline types/resize-observer-browser/test/resize-observer-global.test.ts]: The name of test fioles is missing `-browser` you want your tests to be named as `resize-observer-browser-global.tests.ts` and `resize-observer-browser-module.tests.ts`

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/resize-observer-browser/test/resize-observer-global.test.ts (hunk 2 above)
Comment (chivesrs) [reply types/resize-observer-browser/test/resize-observer-global.test.ts]: Thanks for catching this, fixed.

**Comment by chivesrs** [comment]
Renamed the tests to the expected naming convention.

**Pull request merged**